src/ripple/net/InfoSub.h

```diff
@@ -168,6 +197,12 @@ class InfoSub : public CountedObject<InfoSub>
     void
     deleteSubAccountInfo(AccountID const& account, bool rt);
 
+    bool
```

Comment:
It would be nice to document what the return value means.

Reply:
thanks, added.